apps/web/pages/en/[slug]/index.ts

```diff
@@ -0,0 +1,6 @@
+import withApollo from '@island.is/web/graphql/withApollo'
+import { withLocale } from '@island.is/web/i18n'
+import { withMainLayout } from '@island.is/web/layouts/main'
```

Comment:
I disagree with moving this from the screen components. This is an integral part of the screen itself, and should not be configurable by someone using the component